README.md

```diff
@@ -9,45 +9,65 @@ See [screenshots](#screenshots) below, or jump right to [setup instructions](#se
 
 ## Setup
 
-### Proof General
-
-Download and install Proof General from [GitHub](https://github.com/ProofGeneral/PG):
-
-```bash
-git clone https://github.com/ProofGeneral/PG ~/.emacs.d/lisp/PG
-cd ~/.emacs.d/lisp/PG
-make
-```
-
-Then add the following to your `.emacs`:
-
-```elisp
-;; Open .v files with Proof General's Coq mode
-(load "~/.emacs.d/lisp/PG/generic/proof-site")
-```
-
 ### MELPA
 
-MELPA is a repository of Emacs packages. Skip this step if you already use [MELPA](http://melpa.org/#/getting-started). Otherwise, add the following to your `.emacs` and restart Emacs:
+Both `proof-general` and `company-coq` are on [MELPA](https://melpa.org/#/getting-started), a repository of Emacs packages. Skip this step if you already use MELPA. Otherwise, add the following to your `.emacs`:
 
 ```elisp
 (require 'package)
-(add-to-list 'package-archives '("melpa" . "http://melpa.org/packages/") t)
+(let* ((no-ssl (and (memq system-type '(windows-nt ms-dos))
+                    (not (gnutls-available-p))))
+       (proto (if no-ssl "http" "https")))
+  (add-to-list 'package-archives
+               (cons "melpa" (concat proto "://melpa.org/packages/")) t))
```

Comment:
Let's just use https:// here; that recipe on MELPA is designed to handle all sorts of esoteric configurations, but there's no need to burden this readme with them.

Reply:
Good idea, it'll be simpler with https only.